src/SFA.DAS.Commitments.AddEpaToApprenticeships.WebJob.UnitTests/WhenInitializingContainer.cs

```diff
@@ -0,0 +1,17 @@
+﻿//using NUnit.Framework;
+//using SFA.DAS.Commitments.AddEpaToApprenticeships.WebJob.DependencyResolution;
+
+//namespace SFA.DAS.Commitments.AddEpaToApprenticeships.WebJob.UnitTests
+//{
+//    [TestFixture]
+//    public class WhenInitializingContainer
```

Comment:
Can this be reinstated or removed?

Reply:
I've removed it as it's probably not worth the effort to get it to pass.